src/v/redpanda/admin/api-doc/broker.json

```diff
@@ -7,6 +7,21 @@
         "application/json"
     ],
     "apis": [
+        {
+            "path": "/v1/cluster_view",
+            "operations": [
+                {
+                    "method": "GET",
+                    "summary": "Get cluster view",
+                    "type": "cluster_view",
+                    "nickname": "get_cluster_view",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": []
+                }
+            ]
+        },
```

Comment:
should we deprecate `v1/brokers` in favor of `v1/cluster` which includes the brokers list plus some extra metadata? I would imagine there are only a few callers of the `v1/brokers` endpoint.

Reply:
you're right; but rpk depends on brokers so I wanted to let this PR in first, I've created [an issue](https://github.com/vectorizedio/redpanda/issues/3605) to follow up in rpk